guide/resources/design-research.md

```diff
@@ -1,36 +1,36 @@
 ---
 layout: guide
-title: User research resources
-description: Surveys, academic papers, articles and other resources on bitcoin user research.
-nav_order: 1
-parent: User research
-grand_parent: Designing bitcoin products
-permalink: /guide/designing-products/user-research/resources
+title: Design research
```

Comment:
Interesting decision to switch from `User research` to `Design research`. Trying to figure out if I agree or not...

Reply:
Made more sense to me, some of the research contained is more product focused, some more user behavior focused. I look at them as distinct. Design is a more general term that encompasses both of those.

Reply:
I'm down with it.

Reply:
Agreed